### DISCUSSION TITLE
Distribution of the work in Data Science Project between team members.

### DISCUSSION THREAD

**Discussion by RaofaizanAPSACS** [community/community#31113] [Other Feature Feedback, Questions, & Ideas] [answered]
**How we can distribute or divide the tasks between the team members when working on Data Science Project? Because the steps are dependent on each other.**

**Comment by jfullstackdev** [comment, marked as answer]
you should be able to find a way,

in our development team, the tasks are separated through Modules, each module is a certain page

also, if your design really depends on one another, then it's tightly coupled and a very poor practice in coding, make sure you provide parameters at each function so it will not wait for another function to be created first, but of course even if your team is working separately, the goal later is to build it so that they will work together when you run the actual program

for example, if a certain function will, say, read the temperature first, you don't need to wait for the actual temperature, you can simply create a function with the temperature as parameter, so you can mock it in your testing, that way, you can continuously develop it even without the actual temperature
